tests/pytest/core/test_session.py

```diff
@@ -269,12 +269,12 @@ def test_reset_enrollment(app_request):
 @pytest.mark.django_db
 def test_reset_oauth(app_request):
     app_request.session[session._OAUTH_TOKEN] = "oauthtoken456"
-    app_request.session[session._OAUTH_CLAIM] = "claim"
+    app_request.session[session._OAUTH_CLAIMS] = "claim"
```

Comment:
Similar comment as above: I think this should be a list like `app_request.session[session._OAUTH_CLAIMS] = ["claim"]` to be more aligned with what the value would actually be set to.

Reply:
Definitely agree, thanks! Changed it in c0a2516db02df7dea3467262321d145e5117d8b4